tests/testthat/test-particle.R

```diff
@@ -0,0 +1,88 @@
+context("particle")
+
+test_that("simulation with one particle", {
+  ## TODO: We might write a reference that does not use odin here;
+  ## this is not complicated stuff.
+  gen <- odin::odin({
```

Comment:
Something (perhaps in pure R) with a set RNG sequence might be helpful. This looks good for now though (before gsl is added)